src/compiler/php_generator_helpers.h

```diff
@@ -28,8 +28,11 @@ namespace grpc_php_generator {
 
 inline std::string GetPHPServiceClassname(
     const grpc::protobuf::ServiceDescriptor* service,
-    const std::string& class_suffix) {
-  return service->name() + (class_suffix == "" ? "Client" : class_suffix);
+    const std::string& class_suffix, bool is_server) {
+  return service->name() +
+         (class_suffix == "" ? (is_server ? "Service" : "Client")
```

Comment:
The dilemma here is that, if the service is already named `TestService` for example, then we will be generating the class `TestServiceServiceStub` in `TestServiceServiceStub.php`.

Python would have called it `TestServiceServicer`. C# calls it `TestServiceBase`. Ruby and C++ just call it `TestService`.

I am not sure what the best name should be...